packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -45,8 +57,76 @@ const SelectedPersonaInner = React.memo(
       getExpandedItems,
       styles,
       theme,
+      dragDropHelper,
+      dragDropEvents,
+      eventsToRegister,
     } = props;
     const itemId = getId();
+    const [root, setRoot] = React.useState<HTMLElement | undefined>();
+    const [dragDropSubscription, setDragDropSubscription] = React.useState<IDisposable>();
+    const [isDropping, setIsDropping] = React.useState(false);
+    const [droppingClassNames, setDroppingClassNames] = React.useState('');
+    const [droppingClassName, setDroppingClassName] = React.useState('');
+    const [isDraggable, setIsDraggable] = React.useState<boolean | undefined>(false);
+
+    const _onRootRef = React.useCallback((div: HTMLDivElement) => {
+      if (div) {
+        // Need to resolve the actual DOM node, not the component.
+        // The element itself will be used for drag/drop and focusing.
+        setRoot(ReactDOM.findDOMNode(div) as HTMLElement);
+      } else {
+        setRoot(undefined);
+      }
+    }, []);
+
+    const _updateDroppingState = (newValue: boolean, event: DragEvent) => {
+      if (!newValue) {
+        if (dragDropEvents!.onDragLeave) {
+          dragDropEvents!.onDragLeave(item, event);
+        }
+      } else if (dragDropEvents!.onDragEnter) {
+        setDroppingClassNames(dragDropEvents!.onDragEnter(item, event));
+      }
+
+      if (isDropping !== newValue) {
+        setIsDropping(newValue);
+      }
+    };
+
+    const dragDropOptions: IDragDropOptions = {
+      eventMap: eventsToRegister,
+      selectionIndex: index,
+      context: { data: item, index: index },
+      canDrag: dragDropEvents?.canDrag,
+      canDrop: dragDropEvents?.canDrop,
+      onDragStart: dragDropEvents?.onDragStart,
+      updateDropState: _updateDroppingState,
+      onDrop: dragDropEvents?.onDrop,
+      onDragEnd: dragDropEvents?.onDragEnd,
+      onDragOver: dragDropEvents?.onDragOver,
+    };
+
+    const events = new EventGroup(this);
+
+    function createDragDropSubscription() {
+      return dragDropHelper?.subscribe(root as HTMLElement, events, dragDropOptions);
+    }
+
+    React.useEffect(() => {
+      setDragDropSubscription(createDragDropSubscription());
+      return () => {
+        dragDropSubscription?.dispose();
+        setDragDropSubscription(undefined);
+      };
+    }, [dragDropHelper, dragDropSubscription, createDragDropSubscription]);
+
+    React.useEffect(() => {
+      setIsDraggable(dragDropEvents ? !!(dragDropEvents.canDrag && dragDropEvents.canDrag) : undefined);
```

Comment:
for dragDropEvents.canDrag && dragDropEvents.canDrag.. did you mean to just use one

Reply:
Mean to do canDrag and canDrop. Fixed in the latest iteration.